pkg/envoy/bootstrap/config_test.go

```diff
@@ -115,128 +102,199 @@ static_resources:
         '@type': type.googleapis.com/envoy.extensions.upstreams.http.v3.HttpProtocolOptions
         explicit_http_config:
           http2_protocol_options: {}
+  - load_assignment:
```

Comment:
Thanks for adding tests for the health probes. At some point these YAMLs get unwieldy (I'm guilty of starting this pattern). So we may need to rethink this or chunk it into smaller YAMLs perhaps...